org.gnomeradio.Radio.json

```
@@ -0,0 +1,52 @@
+{
+  "app-id" : "org.gnomeradio.Radio",
+  "runtime" : "org.gnome.Platform",
+  "runtime-version" : "45",
```

```suggestion
  "runtime-version" : "44",
```

There is no GNOME 45 yet.